jahia-cloud-env-update/update-to-3.2.0/update.yml

```
@@ -60,3 +77,15 @@ actions:
         systemctl restart datadog-agent
       fi
     user: root
+    
+  addHTTPSecuityHeaders:
+    cmd[bl]: |-
+      cp /etc/haproxy/haproxy.cfg.d/00-global.cfg /etc/haproxy/haproxy.cfg.d/00-global.cfg.bak
+      insert_line_1='    http-response set-header Strict-Transport-Security "max-age=63072000;"'
+      insert_line_2='    http-response set-header X-Content-Type-Options "nosniff"'
+      insert_line_3='    http-response set-header X-XSS-Protection "1; mode=block;"'
+      pattern='(Host) len 50'
+      sed -ie "/$pattern/a\\\n$insert_line_1\n$insert_line_2\n$insert_line_3" /etc/haproxy/haproxy.cfg.d/00-global.cfg
+      chown -R haproxy: /etc/haproxy
+      service haproxy reload
```

```suggestion
      insert_line_1='    http-response set-header Strict-Transport-Security "max-age=63072000;"'
      insert_line_2='    http-response set-header X-Content-Type-Options "nosniff"'
      insert_line_3='    http-response set-header X-XSS-Protection "1; mode=block;"'
      if ! (grep -qPzo "$insert_line_1\n$insert_line_2\n$insert_line_3" /etc/haproxy/haproxy.cfg.d/00-global.cfg); then
        echo "headers already there"
        exit 0
      fi
      pattern='(Host) len 50'
      sed -i.bak "/$pattern/a\\\n$insert_line_1\n$insert_line_2\n$insert_line_3" /etc/haproxy/haproxy.cfg.d/00-global.cfg
      chown haproxy: /etc/haproxy/00-global.cfg.bak
      service haproxy reload
```
not sure about the `.bak` is relevant here but why not.

the added `grep` is not tested but should be use for  not having several time the headers line if manifest have to be applied several time on an env.